timon/probes/probes.py

```diff
@@ -83,9 +83,9 @@ async def run(self):
             raise
         finally:
             if rsrc:
-                print("RLS RSRC", rsrc)
+                logger.debug("RLS RSRC %r", rsrc)
                 rsrc.semaph.release()
-                print("RLSD RSRC", rsrc)
+                logger.debug("RLSD RSRC %r", rsrc)
```

Comment:
hmmm. this is strange and looks really like somebody (me???) debugging something he didn't understand.

shouldn't `rsrc.semaph.release()` be non blocking in any case?

I would say line 86 is not needed as it will always be followed by line 88, no?

Reply:
line 86 removed